README.md

```diff
@@ -17,6 +17,8 @@ Add your answers to this markdown file.
 
 Answers:
 
-Asymptotic analysis may be misleading for actual performance because it is detailing very large inputs, so a small input size could cause the determining factor of runtime to be one of the constants that gets dropped when performing an asymptotic analysis. The input could be in the best or worst format for the algorithm, such as recieving an already sorted array for either insertion sort or quick sort, which could vastly change runtime. It also gives performance with respect to input values, not actual runtimes.
-Binary search has an average runtime of log(n), so log_2 (1000) = 9.96, and log_2 (10000) = 13.29, about a 50% gain, so an estimated runtime of 7.5 seconds.
-A difference like this could be because the tree had the searched for element in the first position, this is the worst case scenario with the algorithm required to go through every single step to find the result. This could also be due to the case with 1000 elements, if the tree was formatted for best case scenario, which asymptotically does not scale, the 5 seconds could be initial operations before the search begun, which would mean the estimation was based on bad data. The algorithm could also be inefficient and not scale exactly with log(n), an additional coefficient would be unaccounted for in the analysis, which would drastically increase runtime.
+Asymptotic analysis may be misleading for actual performance because it is detailing very large inputs, so a small input size could cause the determining factor of runtime to be one of the constants that gets dropped when performing an asymptotic analysis. An algorithm that has asymptotic complexity of n^5 would be inefficient for inputs of more than 100, but given a single input, the main runtime could actually be controlled be a constant +20 that wouldn't be accounted for in the asymptotic analysis; 100000 log n and 2 log n are both simply log n using this approach. The input could be in the best or worst format for the algorithm, such as recieving an already sorted array for either insertion sort or quick sort, which could vastly change runtime. Additionally, asymptotic analysis assumes a perfect environment, an algorithm optimized for one type of machine or runtime environment may run significantly slower outside of optimized environments. For Example the dual pivot vs single pivot for quicksort in javascript versus python.
```

Comment:
Asymptotic analysis doesn't require very large inputs, just something larger than $n_0$ (which may be 1). Again, we can analyze best/worst case separately; this is only misleading if you analyze the wrong case. In what way does the environment need to be "perfect"?